CHANGELOG.md

```diff
@@ -3,6 +3,15 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [2.2.0] - 2023-04-06
```

Comment:
Let's change this to `Unreleased` as we don't have a definitive version number or release date yet.

```suggestion
## [Unreleased]
```